data/data.js

```diff
@@ -1,73 +1,73 @@
-    "vim": ["1991", "Vim scriptは、1991年にBram Moolenaarによって開発されたプログラミング言語である。", "Vim script"],
-    "vue": ["2014", "Vue.jsは、2014年にEvan Youによって開発されたプログラミング言語である。", "Vue.js"],
-    "wat": ["2018", "watは、2018年にWebAssembly Community Groupによって開発されたプログラミング言語である。", "wat"],
-    "wy": ["2018", "WenyanLangは、2018年にYiyi Wangによって開発されたプログラミング言語である。", "WenyanLang"],
-    "zen": ["2015", "ZenScriptは、2015年にCraftTweakerによって開発されたプログラミング言語である。", "ZenScript"],
-    "zig": ["2015", "Zigは、2015年にAndrew Kelleyによって開発されたプログラミング言語である。", "Zig"],
+    "adb": ["2003", "Androidデバイスと通信するためのコマンドラインツール。デバッグや他の開発目的に利用される", "Android Debug Bridge"],
+    "appleScript": ["1993", " AppleのMac OSで使用される自動化スクリプト言語", "AppleScript"],
+    "async.py": ["2015", "Pythonの非同期プログラミングの一部。非同期I/Oを効率的に行うために使用される", "Async Python"],
+    "bash": ["1989", "Unixシェルとコマンド言語。Unix系OSで広く使用されている。", "Bash"],
+    "bat": ["1989", "Windowsのバッチファイルスクリプト。主に自動化タスクに使用される", "Bat"],
+    "c": ["1972", " 汎用的なプログラミング言語で、OSや組み込みシステムの開発によく使用される", "C"],
+    "clj": ["2007", "Lispに基づいた関数型プログラミング言語で、JVM上で動作する", "Clojure"],
+    "coffee": ["2009", "JavaScriptをより簡潔に書くための言語。JavaScriptにコンパイルされる", "CoffeeScript"],
+    "cpp": ["1983", "C言語にオブジェクト指向プログラミングとジェネリックプログラミングを追加した言語", "C++"],
+    "cr": ["2014", "高性能で型安全なRuby風の言語", "crystal"],
+    "d": ["2001", "C++の代替として設計されたオブジェクト指向プログラミング言語", "D"],
+    "dart": ["2011", "Googleが開発した、クライアント開発に特化した言語。Flutterフレームワークと一緒に使われることが多い", "Dart"],
+    "dash": ["2010", "Unixシェルの一種。シェルスクリプトの実行に特化している", "Dash"],
+    "dpr": ["2018", "/*unknown*/", "DPR"],
+    "exs": ["2011", " Erlang VM上で動作する関数型プログラミング言語 並行処理と分散システムに強い", "elixir"],
+    "elm": ["2012", "フロントエンド開発に使われる純粋関数型プログラミング言語 JavaScriptにコンパイルされる", "elm"],
+    "erl": ["1986", " 並行性と分散システムに焦点を当てた関数型プログラミング言語", "erlang"],
+    "el": ["2018", "/*unknown*/", "EL"],
+    "fs": ["2005", "マイクロソフトが開発した、.NETフレームワーク上で動作する関数型プログラミング言語", "F#"],
+    "fth": ["1970", "スタックベースの低レベルプログラミング言語 組み込みシステムやリアルタイムシステムに使われることが多い", "Forth"],
+    "go": ["2009", "Googleが開発した静的型付けのコンパイル言語 シンプルさと効率性を重視している", "Go"],
+    "groovy": ["2003", "Javaに似た動的なスクリプト言語 Javaよりも短く簡潔なコードを書くことができる", "Groovy"],
+    "hack": ["2014", "Facebookが開発したPHPベースのプログラミング言語 静的型付けと動的型付けを組み合わせている", "Hack"],
+    "hs": ["1990", "高度な純粋関数型プログラミング言語 型推論と遅延評価が特徴", "Haskell"],
+    "html": ["1993", "ウェブページを作成するためのマークアップ言語 ウェブの基礎となる言語", "HTML"],
+    "hx": ["2005", "静的に型付けされた、マルチパラダイムの言語 JavaScriptやC++など多くの言語にコンパイルできる", "Haxe"],
+    "hy": ["2013", "Lispの構文を持つPython上の言語 Pythonのライブラリと互換性を持つ", "Hy"],
+    "java": ["1995", "オブジェクト指向のプログラミング言語 プラットフォームに依存しない特性を持つ", "Java"],
+    "jl": ["2012", " 高性能な数値計算やデータ分析に使われる動的プログラミング言語", "Julia"],
+    "jst": ["1995", "クライアントサイドとサーバーサイドのウェブ開発に広く使われる言語", "JavaScript"],
+    "kn": ["2012", "日本製の教育用プログラミング言語 初心者でも学びやすいことを目指している", "Kuin"],
+    "kt": ["2011", " JVM上で動作する静的型付けのプログラミング言語 Javaと100%互換性がある", "Kotlin"],
+    "lol": ["2007", "ユーモラスなプログラミング言語 インターネットのミーム言語に基づいている", "LOLCODE"],
+    "lsp": ["2018", "/*unkown*/", "LSP"],
```

Comment:
Language Server Protocol じゃん

```suggestion
    "lsp": ["2018", "/*unkown*/", "LSP"],
```

迫真の ***u n k o w n***